boards/librem_mini/librem_mini.config

```diff
@@ -1,3 +1,4 @@
+# WARNING: This system remains perpetually vulnerable to Spectre v2 (CVE-2017-5715). Mitigations and microcode updates previously applied are now known to be ineffective due to QSB-107 and related CVEs. If Spectre v2 is a concern in your threat model, consider migrating to a platform with ongoing microcode support.
```

Comment:
Mini v1 is 8th gen, which [has a microcode update from Intel](https://github.com/intel/Intel-Linux-Processor-Microcode-Data-Files/blob/main/releasenote.md#microcode-20250512).

Reply:
Fixed. This doesn't mean that its applied from firmware though (it is not). A commit fixing 3rdparty/intel-microcode would be needed under purism fork @JonathonHall-Purism